lib/Service/UserConfigService.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2022 John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @author John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Photos\Service;
+
+use Exception;
+use OCA\Photos\AppInfo\Application;
+use OCP\Files\IRootFolder;
+use OCP\IConfig;
+use OCP\IUserSession;
+
+class UserConfigService {
+	public const DEFAULT_CONFIGS = [
```

Comment:
Could theoritecally be exposed as a capability and used straight away by the https://github.com/nextcloud/photos/blob/34948e3f8ad93a30a68cdc0adebf91f642d141ff/src/mixins/UserConfig.js#L35 instead of having to declare them again in the front :)